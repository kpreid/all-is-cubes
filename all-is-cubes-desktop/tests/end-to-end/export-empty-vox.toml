# It is an error if there is nothing to export (at least for now).
# We specifically want to test the export-via-record path, which .vox is an example of.
bin.name = "all-is-cubes"
args = "--no-config-files --template=blank --graphics=record --output=test-export-empty.vox"
status.code = 1
