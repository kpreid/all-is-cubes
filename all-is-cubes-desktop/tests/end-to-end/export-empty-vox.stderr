...
Error: failed to configure session for recording

Caused by:
    universe "blank" contains no default space to export
